src/desktop/apps/home/client/index.coffee

```diff
@@ -37,7 +37,7 @@ module.exports.HomeView = class HomeView extends Backbone.View
     ,
     {
       triggerOnce: true,
-      offset: 75,
+      offset: 500,
```

Comment:
Previously we only fired the experiment viewed event when entry points reached the top of the homepage (within full view). We made this change here so that as soon as the first row of the entry points are in view we fire the event.

Reply:
My only feedback/question was about how this affects the firing in mobile views, but I looked at the Slack discussion that was linked and it looks like they were firing there as expected. Looks good to me!